.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -5,40 +5,43 @@ body:
   - type: checkboxes
     attributes:
       label: Already reported ? *
-      description: Before opening a new bug report, please take a moment to search through the current open and closed issues to check if it already exists.
+      description: Before opening a new bug report, please take a moment to search through the current open issues. If same bug is already reported, don't open new issue - instead go upvote/comment on existing one.
       options:
-      - label: I have searched the existing open and closed issues.
-        required: true
+        - label: I have searched the existing open and closed issues.
+          required: true
 
   - type: dropdown
     id: type
     attributes:
-      label: Regression?
+      label: Is this bug a regression?
```

Comment:
```suggestion
      label: Is this bug due to regression?
```

Reply:
Or maybe "is this issue a regression?" 
Regression may not be necessarily a  bug, e.g. perf degradation, but still someone might want to report it

Reply:
sure, go ahead